packages/dev/core/src/Engines/nativeEngine.ts

```diff
@@ -1636,13 +1636,19 @@ export class NativeEngine extends Engine {
 
         if (!!texture && !!texture._hardwareTexture) {
             const context = canvas.getContext();
-            context.flush();
+            if (context.flush) {
+                context.flush();
+            }
             const source = canvas.getCanvasTexture();
             const destination = texture._hardwareTexture.underlyingResource;
-            this._commandBufferEncoder.startEncodingCommand(_native.Engine.COMMAND_COPYTEXTURE);
-            this._commandBufferEncoder.encodeCommandArgAsNativeData(source as NativeData);
-            this._commandBufferEncoder.encodeCommandArgAsNativeData(destination as NativeData);
-            this._commandBufferEncoder.finishEncodingCommand();
+            if (this._engine.copyTexture) {
+                this._engine.copyTexture(destination, source);
+            } else {
+                this._commandBufferEncoder.startEncodingCommand(_native.Engine.COMMAND_COPYTEXTURE);

```

Comment:
`_native.Engine.COMMAND_COPYTEXTURE` should have a `?` on it also in nativeInterfaces.ts.